server/setup.py

```diff
@@ -58,6 +60,7 @@
     version="1.4.1",
     zip_safe=False,
     entry_points={
+        "console_scripts": ["dive=scripts.cli:cli"],
```

Comment:
We should probably update the README with a really brief section that the CLI exists and how to use it.